### PR TITLE
Add 2.8 release notes

### PR DESCRIPTION
**Pull request by trevorwhitney**
**What this PR does / why we need it**:

Adds 2.8 release notes. Will be copied to https://github.com/grafana/loki/releases/tag/untagged-9c568e6125efad883360 and published upon approval.

### CODE REVIEW

**Review by dannykopping — APPROVED** [review]
LGTM!

File: docs/sources/release-notes/v2-8.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- **Remote rule evaluation** Rules can now be evaluated against a configured `query-frontend`, allowing these queries to be executed with all the parallelisation & optimisation that regular adhoc queries have.
+- **Rule evaluation jitter** Rulers now expose a configurable random sleep before each rule evaluation to prevent contention on the query engine should multiple runs execute concurrently.
+- **Hierarchical scheduler queues** The scheduler queues are now hierarchical, allowing QoS (query fairness) controls within the same tenant via the `X-Actor-Path` header.
+- **Query blocker** Queries can now be blocked in the Querier/Ruler via a per-tenant runtime configuration.
+- **Query max bytes read limit** Two new per-tenant limits were added to protect queriers from reading too much data. Setting `max_query_bytes_read` will refuse queries that will read more than the value configured. Setting `max_querier_bytes_read` will refuse queries whose subqueries would read more than the configured value after splitting and sharding.
```
Comment (liguozhong) [inline docs/sources/release-notes/v2-8.md:17]: `this feature enable only  when TSDB is used.
`

I suggest that we need to add this sentence in the release notes. 

why？
I am looking forward to this feature to limit the current unreasonable recording rules online, but when I upgraded to the latest version after a week, I found that the cassandra indexer I used could not use this feature. I was very disappointed. Work did not get the expected result

File: docs/sources/release-notes/v2-8.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description: Version 2.8 release notes
+weight: 55
+---
+
+# V2.8
+Grafana Labs is excited to announce the release of Loki 2.8. Here's a summary of new enhancements and important fixes:
+
+## Features and enhancements
+
+- **TSDB index no longer experimental** After extensive testing of the new TSDB index in our Grafana Cloud Logs offering, we can safely announce that it is no longer experimental and encourage all Loki deployments to use it!
```
Comment (liguozhong) [inline docs/sources/release-notes/v2-8.md:12]: Great, TSDB is finally ready, I can finally replace Cassandra, operating and maintaining dozens of Cassandra clusters is really painful for me

File: docs/sources/release-notes/v2-8.md (hunk 1 above)
Comment (trevorwhitney) [reply docs/sources/release-notes/v2-8.md:17]: good call out, will do!

**Pull request merged**